packages/math-input/src/store/types.ts

```diff
@@ -2,22 +2,39 @@ import GestureManager from "../components/gesture-manager";
 import VelocityTracker from "../components/velocity-tracker";
 import {LayoutModes} from "../consts";
 
+import type {KeypadType} from "../consts";
 import type {Key} from "../data/keys";
 import type {Cursor, KeyHandler, Popover, Echo} from "../types";
 
+// Interaction between keypad and input
 export interface InputState {
+    // This is the callback to tell the input
+    // that a key was pressed. It returns information
+    // about where the cursor is
     keyHandler: KeyHandler | null;
+    // Information about where the cursor is, which we use to
+    // conditionally render buttons to help navigate
+    // where the cursor should go next
```

Comment:
Awesome. Thanks for adding comments in this file!  Super helpful for me and future others. :)